day_7/10_little_monkeys.rb

```diff
@@ -0,0 +1,24 @@
+puts "--------10 Little Monkeys--------"
+
+
+
+def monkeys(monkeys1, monkeys2, monkeys3 )
+  puts  "#{monkeys1} little monkeys jumping on the bed,
+   One fell off and bumped his head,
+  Mama called the doctor and the doctor said,
+   'No more monkeys jumping on the bed!'
+
+  #{monkeys2} little monkeys jumping on the bed,
+   One fell off and bumped his head,
+   Mama called the doctor and the doctor said,
+   'No more monkeys jumping on the bed!'
+
+   #{monkeys3} little monkey jumping on the bed,
+   He fell off and bumped his head,
+   Mama called the doctor and the doctor said,
+    'Get those monkeys right to bed!' "
+end
+
+
+monkeys("Ten", "Nine", "Eight")
```

Comment:
When you see repetitive strings, it's a good sign you could use iteration (probably with an enumerable). Before mod 1, I'd recommend you to try this exercises again where it counts down from "Ten" to "One" monkeys using iteration.